src/System.Private.CoreLib/src/System/Threading/SemaphoreSlim.cs

```diff
@@ -97,8 +86,6 @@ void IThreadPoolWorkItem.ExecuteWorkItem()
                 bool setSuccessfully = TrySetResult(true);
                 Debug.Assert(setSuccessfully, "Should have been able to complete task");
             }
-
-            void IThreadPoolWorkItem.MarkAborted(ThreadAbortException tae) { /* nop */ }
```

Comment:
This is still part of the interface and can't be removed.  Does this compile?

Reply:
Yes it compiles perfectly fine. should I add it again ?

Reply:
> should I add it again ?

Yes, it should be added back.  I expect it's picking up the base type's implementation of the interface, and that's why it's still compiling successfully without this, but in doing so removing this line is potentially changing the behavior here (even though thread aborts are deprecated and in practice we shouldn't be hitting them outside of a debugging environment).